src/Faker/Container/Container.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Faker\Container;
+
+use Psr\Container\ContainerInterface;
+
+/**
+ * A simple implementation of a container.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ * @experimental This class is experimental and does not fall under our BC promise
+ */
+final class Container implements ContainerInterface
+{
+    /**
+     * @var array<string, callable|string|object>
+     */
+    private $definitions = [];
+    private $services = [];
+
+    public function __construct(array $container)
+    {
+        $this->definitions = $container;
+    }
+
+    /**
+     * @throws NotInContainerException
+     * @throws ContainerException
+     */
+     public function get($id)
+    {
+        if (!is_string($id)) {
+            throw new \LogicException(sprintf('First argument of %s::get() must be string', __CLASS__));
+        }
+
+        if (isset($this->services[$id])) {
```

Comment:
```suggestion
        if (array_key_exists($id, $this->services)) {
```

Reply:
If the $this->services[$id] somehow is null, we really want to instantiate it again.

Reply:
Why would it not be null the second time?

Reply:
Im not sure, but if it ever was, I want to instantiate it again. 

But if we believe it never will be null, then why this change?